<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Adminwindow</class>
 <widget class="QWidget" name="Adminwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>395</x>
     <y>22</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Adminwindow/src/RETURN_X_W.png)}
QPushButton:hover{border-image: url(:/Adminwindow/src/RETURN_X_P.png)}
QPushButton:pressed{border-image:url(:/Adminwindow/src/RETURN_X_C.png) }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>70</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">selection-color: rgb(255, 255, 255);
selection-background-color: rgb(0, 0, 0);
background-color: rgb(55,55, 55);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_chooseuser">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>70</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="index.qrc">:/Adminwindow/src/CHOOSEUSER_100_30.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_name">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="index.qrc">:/Adminwindow/src/NAME_100_30.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_username">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>160</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_text_code">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>200</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="index.qrc">:/Adminwindow/src/CODE_100_30.png</pixmap>
   </property>
  </widget>
  <widget class="QLabel" name="label_usercode">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>240</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>340</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{border-image:url(:/Adminwindow/src/DELETE_100_30.png)}
QPushButton:hover{border-image: url(:/Adminwindow/src/DELETE_100_30.png)}
QPushButton:pressed{border-image:url(:/Adminwindow/src/DELETE_100_30.png) }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="index.qrc"/>
 </resources>
 <connections/>
</ui>
